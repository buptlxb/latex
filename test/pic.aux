\relax 
\ifx\hyper@anchor\@undefined
\global \let \oldcontentsline\contentsline
\gdef \contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global \let \oldnewlabel\newlabel
\gdef \newlabel#1#2{\newlabelxx{#1}#2}
\gdef \newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\let \contentsline\oldcontentsline
\let \newlabel\oldnewlabel}
\else
\global \let \hyper@last\relax 
\fi

\@writefile{toc}{\contentsline {section}{\numberline {1}Sec 1}{1}{section.1}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces something}}{1}{figure.1}}
\newlabel{fig:1}{{1}{1}{something\relax }{figure.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Sec 2}{2}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Subsec 2.1}{2}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.1}Subsubsec 2.1.1}{3}{subsubsection.2.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{\numberline {2.1.2}Subsubsec 2.1.2}{4}{subsubsection.2.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Subsec 2.2}{5}{subsection.2.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Sec 3}{6}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}Subsec 3.1}{6}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}Subsec 3.2}{6}{subsection.3.2}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Sec 4}{7}{section.4}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Sec 5}{8}{section.5}}
\@writefile{toc}{\contentsline {section}{\numberline {6}Sec 6}{9}{section.6}}
