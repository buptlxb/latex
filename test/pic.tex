\documentclass[a4paper, 11pt]{article}
\usepackage{graphicx}
\usepackage[colorlinks=true]{hyperref}
\title{Pic}
\author{ict-lxb \and bupt-lxb}
\begin{document}
\maketitle
%\tableofcontents

\section{Sec 1}

\begin{figure}[htbp]
    \centering\includegraphics[width=3.5in]{figures/b_mflops.png}
\caption{something}\label{fig:1}
\end{figure}

text content
\newpage
\section{Sec 2}
\subsection{Subsec 2.1}
\newpage
\subsubsection{Subsubsec 2.1.1}
\newpage
\subsubsection{Subsubsec 2.1.2}
\newpage
\subsection{Subsec 2.2}
\newpage
\section{Sec 3}
\subsection{Subsec 3.1}
\subsection{Subsec 3.2}
\newpage
\section{Sec 4}
\newpage
\section{Sec 5}
\newpage
\section{Sec 6}
\newpage
\end{document}
