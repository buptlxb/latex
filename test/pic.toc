\contentsline {section}{\numberline {1}Sec 1}{1}{section.1}
\contentsline {section}{\numberline {2}Sec 2}{2}{section.2}
\contentsline {section}{\numberline {3}Sec 3}{2}{section.3}
\contentsline {section}{\numberline {4}Sec 4}{2}{section.4}
\contentsline {section}{\numberline {5}Sec 5}{2}{section.5}
\contentsline {section}{\numberline {6}Sec 6}{2}{section.6}
