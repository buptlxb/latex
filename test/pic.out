\BOOKMARK [1][-]{section.1}{Sec 1}{}
\BOOKMARK [1][-]{section.2}{Sec 2}{}
\BOOKMARK [2][-]{subsection.2.1}{Subsec 2.1}{section.2}
\BOOKMARK [3][-]{subsubsection.2.1.1}{Subsubsec 2.1.1}{subsection.2.1}
\BOOKMARK [3][-]{subsubsection.2.1.2}{Subsubsec 2.1.2}{subsection.2.1}
\BOOKMARK [2][-]{subsection.2.2}{Subsec 2.2}{section.2}
\BOOKMARK [1][-]{section.3}{Sec 3}{}
\BOOKMARK [2][-]{subsection.3.1}{Subsec 3.1}{section.3}
\BOOKMARK [2][-]{subsection.3.2}{Subsec 3.2}{section.3}
\BOOKMARK [1][-]{section.4}{Sec 4}{}
\BOOKMARK [1][-]{section.5}{Sec 5}{}
\BOOKMARK [1][-]{section.6}{Sec 6}{}
